thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -41,14 +41,23 @@ func GetEvalFnRunner(ctx context.Context, parent string) *EvalFnRunner {
 		RunE:    r.runE,
 		PreRunE: r.preRunE,
 	}
-
 	r.Command = c
 	r.Command.Flags().StringVarP(&r.Dest, "output", "o", "",
 		fmt.Sprintf("output resources are written to provided location. Allowed values: %s|%s|<OUT_DIR_PATH>", cmdutil.Stdout, cmdutil.Unwrap))
 	r.Command.Flags().StringVarP(
 		&r.Image, "image", "i", "", "run this image as a function")
 	_ = r.Command.RegisterFlagCompletionFunc("image", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
-		return cmdutil.FetchFunctionImages(), cobra.ShellCompDirectiveDefault
+		return cmdutil.SuggestFunctions(cmd), cobra.ShellCompDirectiveDefault
+	})
+	r.Command.Flags().StringArrayVarP(
+		&r.Keywords, "keywords", "k", []string{}, "discover the keywords correlated functions")
```

Comment:
I'm having hard time figuring out the help text. I am also having even harder time coming up with a better suggestion :)

my best try: "filter functions that match one or more keywords"

Reply:
quite possibly. IMO we need to (find a way) to communicate that the keywords here are used to filter functions, using the keywords stored on the functions.

(hopefully better try than my previous one): "Filter function; include functions whose keywords match one or more of the keywords provide in this flag"

(may be too wordy though; perhaps we can trim it down)